neurow/lib/neurow/jwt_auth_plug.ex

```diff
@@ -124,13 +125,16 @@ defmodule Neurow.JwtAuthPlug do
     end)
   end
 
-  defp jwt_token_from_request(conn) do
-    case conn |> get_req_header("authorization") do
-      ["Bearer " <> jwt_token] ->
-        {:ok, jwt_token}
-
-      _ ->
-        {:error, :invalid_authorization_header, "Invalid authorization header"}
+  defp jwt_token_from_request(conn, options) do
+    Enum.find_value(options.auth_header_keys, fn header ->
+      case get_req_header(conn, header) do
+        ["Bearer " <> jwt_token] -> jwt_token
```

Comment:
Question: Is this also a Bearer with `x-interservice-authorization` ?

Reply:
Yes, same format, just different name